docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,142 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+<p className="subtitle">Learn how to ignore specific files or folders in your codebase when using Cody.</p>
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+The Cody Ignore feature helps you control what context from your codebase is used by Cody. You can do it via:
 
-## Enabling Unstable Features
+-   Cody Context Filters (Cody Enterprise Only)
+-   Cody Ignore File (All Cody users)
+
+## Cody Context Filters 
+<Callout type="info"> Cody Context Filters is available for Enterprise users and is currently in the Experimental stage.</Callout>
+Admins on the Sourcegraph Enterprise instance can use the **Cody Context Filters** to determine which repositories Cody can use as the context in its requests to third-party LLMs. This functionality is currently experimental and is only supported with the following requirements:
+
+- A valid Cody Enterprise license and a running Sourcegraph version `>=5.4.0`
+- Works if you're using Cody for VS Code, Cody for JetBrains, and Cody for Web as your client
+- Your Cody client meets the following version specs: VS Code `>=1.20.0` and JetBrains `>=6.0.0`
+- Setting the `cody-context-filters-enabled` feature flag to `true`
+
+Administrators can configure the `cody.contextFilters` field in the site configuration with the repos that they wish Cody to exclude or only include using the following structure:
+
+```json5
+{
+	// Optional. If defined, it must have at least one
+	// field set (either `include` or `exclude`).
+	'cody.contextFilters': {
+		// Optional. If defined, it must have at least one item.
+		include: [
+			{
+				// Required. Should follow the RE2 syntax.
+				repoNamePattern: '^github\\.com\\/sourcegraph\\/.*'
+			}
+		],
+		// Optional. If defined, it must have at least one item.
+		exclude: [
+			{
+				// Required. Should follow the RE2 syntax.
+				repoNamePattern: '.*analytics.*'
```

Comment:
>Should follow the RE2 syntax

☝🏻 is an important requirement and it may be overlooked if it's only mentioned in the comment. Should we duplicate it (or move to) as a separate paragraph below the code block (similar to the search page), @MaedahBatool?